element-template-generator/congen-cli/src/main/java/io/camunda/connector/generator/cli/command/Generate.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.connector.generator.cli.command;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.camunda.connector.generator.api.CliCompatibleTemplateGenerator;
+import io.camunda.connector.generator.cli.GeneratorServiceLoader;
+import io.camunda.connector.generator.dsl.ElementTemplateBase;
+import java.util.List;
+import java.util.concurrent.Callable;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Parameters;
+import picocli.CommandLine.ParentCommand;
+
+@Command(name = "generate")
+public class Generate implements Callable<Integer> {
+
+  @ParentCommand ConGen connectorGen;
+
+  @Parameters(
+      index = "0..*",
+      description =
+          "parameters to be passed to the generator (at least the generation source)."
+              + " Refer to the documentation of the specific generator module for details.")
+  List<String> params;
+
+  private static final ObjectMapper mapper = new ObjectMapper();
+
+  @SuppressWarnings("unchecked")
+  @Override
+  public Integer call() {
+    CliCompatibleTemplateGenerator<Object, ?> generator =
+        (CliCompatibleTemplateGenerator<Object, ?>) loadGenerator(connectorGen.generatorName);
+    Object input;
+    try {
+      input = generator.prepareInput(params);
+    } catch (Exception e) {
+      System.err.println("Error while preparing input data: " + e.getMessage());
+      return -1;
```

Comment:
Thought (maybe not for this PR, but future consideration): error codes should be unique. This is not critical here but if at some point, somebody decides to write bash script on top of CLI, they might have issues with identifying exact errors.

Reply:
I set the following exit codes for now: 1 for execution exception, 2 for invalid input. These are defaults offered by the picocli framework.